bin/youclidparser

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/sh
```

Comment:
This isn't the cleanest, but it means that no matter what version of python the user has as their default python3, python3.6 will be run. There's probably a better way to do this where we could just have this file symlink to `main_parser.py` (maybe `#!python3.6` at the top of `main_parser.py` will force python3.6 to run?)